src/redux/selectors/exposedSelectors.js

```diff
@@ -48,6 +48,16 @@ export const isDocumentLoaded = state => state.viewer.isDocumentLoaded;
 export const isDocumentReadOnly = state => state.viewer.isReadOnly;
 export const getCustomPanels = state => state.viewer.customPanels;
 export const getPageLabels = state => state.viewer.pageLabels;
+export const getColorPalette = (state, annotationType) => {
+  if (annotationType !== null){
+    return state.viewer.colorPalette[annotationType].currentPalette;
+  }
+};
+export const getDefaultColorPalette = (state, annotationType) => {
```

Comment:
Maybe I overlooked something, but this selector doesn't seem to be used. Is there any reason why we are having this selector? I can see `getColorPalette` is being used in `StylePopup`

Reply:
I will be using getDefaultColorPalette to fetch the palette colour to be displayed on button icons.